torchvision/ops/boxes.py

```diff
@@ -112,8 +113,19 @@ def clip_boxes_to_image(boxes, size):
     boxes_x = boxes[..., 0::2]
     boxes_y = boxes[..., 1::2]
     height, width = size
-    boxes_x = boxes_x.clamp(min=0, max=width)
-    boxes_y = boxes_y.clamp(min=0, max=height)
+
+    if torchvision._is_tracing():
+        height = height.to(torch.float32)
+        width = width.to(torch.float32)
```

Comment:
I believe this will break torchscript compatibility, because it would imagine that `height` and `width` are int, which do not have a `.to` method.

Would it be possible to make `clamp` work with dynamic values? In PyTorch, `.clamp` already support tensor values for `min` / `max`.

Reply:
I believe clamp only supports constants for min/max:
def clamp(input, min_, max_):
    # type: (Tensor, float, float) -> Tensor

Would this cause a failure with branch:
if torchvision._is_tracing():